react/features/filmstrip/components/web/RaisedHandIndicator.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import BaseIndicator from './BaseIndicator';
+
+/**
+ * Thumbnail badge showing that the participant would like to speak.
+ *
+ * @extends BaseIndicator
+ */
+class RaisedHandIndicator extends BaseIndicator {
+    /**
+     * {@code RaisedHandIndicator} component's property types.
+     *
+     * @static
+     */
+    static propTypes = {
+        /**
+         * The font-size for the icon.
+         *
+         * @type {number}
+         */
+        iconSize: React.PropTypes.number
```

Comment:
Correct me if I'm wrong but on every re-render of the indicators we calling `UIUtil.getIndicatorFontSize();` to get the value of `iconSize` prop. My proposal was to remove the prop `iconSize` and just call the `UIUtil.getIndicatorFontSize();` function in the render method of RaiseHandIndicator.